internal/application/generator.go

```diff
@@ -212,7 +212,7 @@ func (g *Generator) clean(app *redskyappsv1alpha1.Application) error {
 func (g *Generator) readScenario() (*redskyappsv1alpha1.Scenario, error) {
 	// Make sure the file exists
 	if _, err := os.Lstat(g.ScenarioFile); err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
Not sure I understand the change; if the scenario file doesn't exist why wouldn't we want to fail?

Reply:
This is just a blanket "ignore everything" instead of doing something like `if g.ScenarioFile == "" { return nil, nil }` first.